test/api_integration/apis/data_views/fields_for_wildcard_route/response.ts

```diff
@@ -249,11 +249,28 @@ export default function ({ getService }: FtrProviderContext) {
         .get(FIELDS_FOR_WILDCARD_PATH)
         .set(ELASTIC_HTTP_VERSION_HEADER, INITIAL_REST_VERSION_INTERNAL)
         .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana')
-        .query({ pattern: 'basic_index' })
+        .query({ pattern: 'basic_index', allow_no_index: true })
+        .expect((response) => {
+          if (response.statusCode !== 200) {
+            log.debug(response.body);
+          }
+        })
         .expect(200, {
           fields: [],
           indices: [],
         });
+
+      await supertest
```

Comment:
Can we make this a new test since the description for the existing one is `returns 200 when index is closed`?

Reply:
@lukasolson Sure, I split them in 2 tests https://github.com/elastic/kibana/pull/199717/commits/ffa49aac9824fa00638ea5bf456cce13dd77b265, thanks!